rebasehelper/plugins/output_tools/__init__.py

```diff
@@ -51,8 +51,13 @@ def get_report_path(cls, app):
         return os.path.join(app.results_dir, REPORT + '.' + cls.EXTENSION)
 
     @classmethod
-    def prepend_results_dir_name(cls, *path_members):
-        return os.path.join(RESULTS_DIR, *path_members)
+    def prepend_results_dir_name(cls, app, *path_members):
+        if app.conf.results_dir:
+            # revert back from path relative to execution dir to path
+            # relative to starting dir
+            prepend = os.path.relpath(os.path.join(app.execution_dir, app.conf.results_dir), app.start_dir)
```

Comment:
What if `results_dir` is absolute? Also, I don't like the expression `revert back`, even though it is technically correct. I think `make the path relative to start_dir` would be enough. You can add `(as opposed to being relative to execution_dir)` if you think it's necessary to mention that.

Reply:
If `results_dir` is absolute, it correctly makes the paths relative to the current directory. The output is correct and makes sense but perhaps it would be better to use absolute paths if the user used absolute path...

Reply:
But you are prepending absolute `results_dir` with `execution_dir`. Maybe it really does work, I didn't try it, but it definitely looks wrong. Anyway, yes, if `results_dir` was specified as absolute, it should be absolute in the output as well.